contentful/export-processor/options-helpers.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Get argument value from options or environment
+ * @param {object} obj - Object containing arguments (e.g. parsed CLI arguments or process.env)
+ * @param {string} key - The key of the option
+ * @param {boolean} isBoolean - Whether the value should be treated as a boolean
+ * @returns {*} Value based on the options or environment, or undefined if not found
+ */
+const getArgumentValue = (obj, key, isBoolean) => {
```

Comment:
@jimwashbrook did you mean to do this on this branch?

Reply:
Oui; quick dirty fix for the error caused by other changes

Reply:
Yes, he was fixing the issue caused by resolving the merge conflict